pages/common/drill.md

```diff
@@ -0,0 +1,31 @@
+# drill
+
+> Get information out of DNS.
```

Comment:
Perhaps `Perform various DNS queries.` would be a better description here?

Reply:
FWIW - we deliberately try to avoid picking up lines verbatim from the man page. Otherwise a user can just look up the man page itself instead of checking the tldr page.